### PR TITLE
Add Ruby 2.6.1

### PR DESCRIPTION
**Pull request by alyssais**
I also threw in some other changes that were useful to me in adding the version.

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ case "$ruby" in
 	rubinius)
 		ruby="rbx"
 		exts=(tar.bz2)
-		downloads_url="http://releases.rubini.us/"
+		downloads_url="https://releases.rubini.us/"
```
Comment (eregon) [inline update.sh]: Does this work though?

File: update.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ case "$ruby" in
 		version_family="${version:0:3}"
 
 		exts=(tar.gz tar.bz2 tar.xz zip)
-		downloads_url="http://cache.ruby-lang.org/pub/ruby/"
+		downloads_url="http://cache.ruby-lang.org/pub/ruby"
```
Comment (eregon) [inline update.sh]: Actually all `downloads_url` should be without the final `/`, as there is always a `/` added later on.
Could you remove it also in other places?

File: update.sh (hunk 1 above)
Comment (alyssais) [reply update.sh]: Turns out Rubinius support was just completely broken. I've fixed it now.

File: update.sh (hunk 2 above)
Comment (alyssais) [reply update.sh]: Done!

**Comment by havenwood** [comment]
Thank you for the PR!

File: update.sh (hunk 2 above)
Comment (eregon) [reply update.sh]: Thanks!

**Pull request merged**